### PR TITLE
fix: remove incorrect min-width values from select and button components

### PR DESCRIPTION
**Pull request by chrisdholt**
## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-components-styles-msft/src/button/index.ts [hunk 1]
```diff
@@ -120,7 +120,7 @@ const styles: ComponentStyles<ButtonClassNameContract, DesignSystem> = {
         maxWidth: "374px",
         minWidth: ensureDesignSystemDefaults(
             (designSystem: DesignSystem): string =>
-                designSystem.density <= -2 ? "100px" : "120px"
+                designSystem.density <= -2 ? "28px" : "32px"
```
Comment (nicholasrice) [inline packages/fast-components-styles-msft/src/button/index.ts]: should this be a function of the design unit?

File: packages/fast-components-styles-msft/src/select/index.ts [hunk 2]
```diff
@@ -24,24 +24,21 @@ import { inputFieldStyles } from "../patterns/input-field";
 
 const styles: ComponentStyles<SelectClassNameContract, DesignSystem> = {
     select: {
-        minWidth: "276px",
+        display: "inline-block",
```
Comment (nicholasrice) [inline packages/fast-components-styles-msft/src/select/index.ts]: What was this before - block? Are we worried about breaking layouts changing this?

File: packages/fast-components-styles-msft/src/button/index.ts (hunk 1 above)
Comment (chrisdholt) [reply packages/fast-components-styles-msft/src/button/index.ts]: It isn't anywhere else, so I'm a bit hesitant to swap the logic out for something more dynamic right now. Considering this pattern exists throughout the code, I'd prefer we make a change like that in a new PR after consulting with @bheston who drove this as part of the density changes.

File: packages/fast-components-styles-msft/src/select/index.ts (hunk 2 above)
Comment (chrisdholt) [reply packages/fast-components-styles-msft/src/select/index.ts]: @nicholasrice I think that's a fair point; I can remove this change if we want to hold off on this update until vNext

**Pull request merged**